R/tidy-upkeep.R

```diff
@@ -111,6 +111,42 @@ if (year <= 2022) {
     )
   }
 
+  if (year <= 2023) {
+
+    desc <- proj_desc()
+
+    bullets <- c(bullets,
+     "2023",
+     "",
+     "Posit updates:",
+     "",
+     todo('Update copyright holder in DESCRIPTION: `person(given = "Posit, PBC", role = c("cph", "fnd"))`',
```

Comment:
Should look like this https://style.tidyverse.org/syntax.html#long-lines, which is easy to do with http://github.com/lionel-/codegrip